### ISSUE TITLE
UI request: Could we make Path titles persist across steps?

### ISSUE THREAD

**Issue by absempere** [performant-software/gbof-astro#31]
I had several Paths open in separate tabs and was partway through them when I realized I had no idea which Path was which. Parts of the titles are visible in the browser tabs, of course, but it'd be helpful to have the Path title persist across all of the steps. I can easily imagine an instructor in the midst of grading in a similar situation -- except with an entire class's worth of Paths open!

**Comment by jamiefolsom** [comment]
I agree with this one, @absempere -- @cblissg do you have a preference for how we would persist the Path title? Doesn't have to be h1 sized -- could be more subtle.

https://github.com/performant-software/gbof-astro/assets/882377/b0781ad3-ac14-49d5-86b5-0d0512bb1c72
